packages/web-service/src/pages/habitat/a24/work-end/habitat-work-end.js

```diff
@@ -39,8 +39,17 @@ export const setData = async request => {
   const pageData = await request.cache().getPageData()
   const journeyData = await request.cache().getData()
 
-  const day = pageData.payload['habitat-work-end-day']
-  const month = pageData.payload['habitat-work-end-month']
+  let day = pageData.payload[`${habitatURIs.WORK_END.page}-day`]
+  let month = pageData.payload[`${habitatURIs.WORK_END.page}-month`]
+
+  if (day.length === 1) {
```

Comment:
Why not just: 
const endDate = `${year.padStart(2, '0')}-${month.padStart(2, '0')}-${day.padStart(2, '0')}`

Reply:
Just a suggestion. If you want to effect pageData.payload then keep it as is.